packages/jest-reporters/src/notify_impl.ts

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+export default getNotifier();
```

Comment:
could we lazy-load this? Previously it was only required on usage (via a babel plugin), while now we'll require it on startup

Reply:
Can you show me the babel config?

Reply:
https://github.com/facebook/jest/blob/116303baf711df37304986897582eb8078aa24d8/scripts/build.js#L152

Reply:
Essentially what I want is just

```ts
let notifier: typeof import('node-notifier') | null = null;

function loadNotifier() {
  if (!notifier) {
    notifier = require('node-notifier');
  }
  if (!notifier) {
    throw new Error('Unable to load `node-notifier`');
  }

  return notifier;
}
```
or something like that